app/templates/crate/delete.hbs

```diff
@@ -0,0 +1,60 @@
+<div local-class="wrapper">
+  <div local-class="content">
+    <h1 local-class="title" data-test-title>Delete the {{@model.name}} crate?</h1>
+
+    <p>Are you sure you want to delete the crate "{{@model.name}}"?</p>
+
+    <div local-class="warning">
+      {{svg-jar "triangle-exclamation"}}
+      <p><strong>Important:</strong> This action will permanently delete the crate and its associated versions. Deleting a crate is&nbsp;irreversible!</p>
+    </div>
+
+    <div class="impact">
+      <h3>Potential Impact:</h3>
+      <ul>
+        <li>Users will no longer be able to download this crate.</li>
+        <li>Any dependencies or projects relying on this crate will be broken.</li>
+        <li>Deleted crates cannot be reinstated.</li>
+      </ul>
+    </div>
+
+    <div class="requirements">
+      <h3>Requirements:</h3>
+      <p>A crate can only be deleted if:</p>
+      <ul>
+        <li>the crate has been published for less than 72 hours, or</li>
+        <li>the crate only has a single owner,</li>
+        <li>the crate has been downloaded less than 100 times for each month it has been published,</li>
+        <li>and the crate is not depended upon by any other crate on crates.io.</li>
+      </ul>
```

Comment:
I think we're going to need to make the associativity more obvious:

```suggestion
      <ol>
        <li>the crate has been published for less than 72 hours, or</li>
        <li>
          <ul>
            <li>the crate only has a single owner, and</li>
            <li>the crate has been downloaded less than 100 times for each month it has been published, and</li>
            <li>the crate is not depended upon by any other crate on crates.io.</li>
          </ul>
        </li>
      </ol>
```

Which would also need this bit in `delete.module.css`:

```css
.requirements {
    > ol > li:first-child {
        margin-bottom: 0.5em;
    }

    ol ul {
        list-style: none;
        padding-left: 0;
    }
}
```

Which then looks like:

![image](https://github.com/user-attachments/assets/2a30b570-ad2a-4b46-868c-f6672faabcf5)